e2e/rbd.go

```diff
@@ -513,6 +513,7 @@ var _ = Describe("RBD", func() {
 			})
 
 			By("reattach the old PV to a new PVC and check if PVC metadata is updated on RBD image", func() {
+				reattachPVCNamespace := "reattach-cephfs"
```

Comment:
Can you take the current name of the namespace , and append it with `-2` or something instead?